pushjack/apns.py

```diff
@@ -372,10 +389,7 @@ def check_error(self, timeout=10):
 
         if command != APNS_ERROR_RESPONSE_COMMAND:  # pragma: no cover
             self.close()
-            raise APNSServerError(('Error response command must be {0}. '
```

Comment:
Reason for removing this?

Reply:
The exception would be added to the errors list in the sendall method, which would then be passed to the APNSResponse class, which expects all error objects to have the token index in the 'identifier' ivar.